tests/test_utils.py

```diff
@@ -1,7 +1,7 @@
 import pytest
-from mock import Mock, call, patch
+from mock import ANY, Mock, call, patch
 
-from pganonymizer.utils import get_connection, truncate_tables
+from pganonymizer.utils import get_connection, truncate_tables, import_data
```

Comment:
Could you please re-order the import for this line alphabetically? This would be the only annotation for this PR. Thanks again and nice work :+1: